ui/helpers/constants/settings.js

```diff
@@ -342,6 +342,13 @@ export const SETTINGS_CONSTANTS = [
     route: `${EXPERIMENTAL_ROUTE}#autodetect-nfts`,
     icon: 'fa fa-flask',
   },
+  {
+    tabMessage: (t) => t('experimental'),
+    sectionMessage: (t) => t('securityAlerts'),
```

Comment:
Searching for `Security Alerts` in the settings is navigating to the end of the page instead of the correct entry.

Reply:
I don't think it's scrolling to the end of the page, it's just scrolling until the checkbox is in view.

You can observe similar behaviour if you search for currency conversion

Reply:
Initially, I was testing it was scrolling to the end of the page, but now it works correctly.

Reply:
The scrolling issues do exist in some of the settings entries; I think [#19771](https://github.com/MetaMask/metamask-extension/pull/19771) will be addressing them.